sample/src/main/res/layout/activity_main_sample.xml

```diff
@@ -114,13 +114,27 @@
         android:id="@+id/export_to_file"
         android:layout_width="0dp"
         android:layout_height="?attr/actionBarSize"
+        android:layout_marginEnd="@dimen/doub_grid_size"
         android:text="@string/export_to_file"
         app:layout_constraintBottom_toBottomOf="parent"
-        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintEnd_toStartOf="@+id/export_to_file_har"
+        app:layout_constraintHorizontal_bias="0.5"
         app:layout_constraintStart_toStartOf="parent"
         app:layout_constraintTop_toBottomOf="@+id/launch_chucker_directly"
         app:layout_constraintWidth_max="@dimen/max_width" />
 
+    <com.google.android.material.button.MaterialButton
```

Comment:
This is a nit: You haven't update the `layout-land` equivalent for this Layout. I just noticed that also the export to file was missing for that.

If you're fine adding both, that would be gold.
The catch is that there might be not enough space on landscape to add more buttons, so it's probably also fine to just don't add them.

Aside from this, code looks good to me. Please add a screenshot as @vbuberen suggested.

Reply:
@cortinico I've added a couple of screenshot from the sample APP. I went ahead and added the buttons on the landscape layout.